hydra-node/json-schemas/api.yaml

```diff
@@ -141,6 +141,22 @@ channels:
           type: request
           method: POST
           bindingVersion: '0.1.0'
+  /protocol-parameters:
+    description: Endpoint to obtain protocol parameters used in hydra-node.
```

Comment:
Could: be more precise here and say these are the parameters which are used by the ledger in hydra-node.